components/content-service/pkg/service/usage-report-service.go

```diff
@@ -42,20 +42,64 @@ func (us *UsageReportService) UploadURL(ctx context.Context, req *api.UsageRepor
 	span.SetTag("name", req.Name)
 	defer tracing.FinishSpan(span, &err)
 
-	err = us.s.EnsureExists(ctx, us.bucketName)
+	if req.GetName() == "" {
+		return nil, status.Error(codes.InvalidArgument, "Name is required but got empty.")
+	}
+
+	logger := log.WithField("name", req.Name).
```

Comment:
usually we just name the variable `log` in cases like this. 
```suggestion
	log := log.WithField("name", req.Name).
```

Reply:
`log` is the package name where the logging is implemented. It's a name clash that we shouldn't allow.